src/basic/capability-util.c

```diff
@@ -113,16 +114,12 @@ int capability_ambient_set_apply(uint64_t set, bool also_inherit) {
         int r;
 
         /* Remove capabilities requested in ambient set, but not in the bounding set */
-        for (unsigned i = 0; i <= cap_last_cap(); i++) {
-                if (set == 0)
-                        break;
-
+        for (unsigned i = 0; i <= cap_last_cap() && set > 0; i++)
                 if (FLAGS_SET(set, (UINT64_C(1) << i)) && prctl(PR_CAPBSET_READ, i) != 1) {
                         log_debug("Ambient capability %s requested but missing from bounding set,"
-                                        " suppressing automatically.", capability_to_name(i));
+                                  " suppressing automatically.", capability_to_name(i));
```

Comment:
I prefer to not break error message into multiple lines, even if the line exceeds the 109ch limit.
Then, we can easily grep code by the log message.